src/HW_models/nrf.h

```diff
@@ -0,0 +1,11 @@
+#ifndef NRF_H__
```

Comment:
Maybe better place it in a new src/nrfx/mdk/ folder to mimic the folder structure of the real nrfx repo (somebody may include this headers as "mdk/nrf.h" or so)
And add an extra line in the cmake file zephyr_include_directories for this new directory